# Define the goal
geometry_msgs/Pose[] box_poses
---
# Define the result
int32[] box_order          # picking order based on position in the box vector
---
# Define a feedback message
# Nothing